app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-default">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+    <span class="sr-only">Toggle navigation</span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    </button>
+    <%= link_to(root_path, class: 'navbar-brand') do %>
+      <%= image_tag('lvm_circle.png') %>
+      <span>Literacy Volunteers of MA</span>
+    <% end %>
+  </div>
+  <div class="collapse navbar-collapse navbar-right">
+    <ul class="nav navbar-nav">
+      <% if current_user %>
+        <%= render 'layouts/navigation_links' %>
+      <% else %>
+        <li class="dropdown"><%= link_to "Login", new_user_session_path %></li>
+        <li class="dropdown"><%= link_to "Sign Up", new_user_registration_path %></li>
```

Comment:
Might as well remove this; Roberta said she was opting for the "admins create all users" method of user administration.